Makefile

```diff
@@ -7,7 +7,6 @@ PROVIDER_PATH := provider/v4
 VERSION_PATH := $(PROVIDER_PATH)/pkg/version.Version
```

Comment:
Changes in this file are expected since I picked up the failed `ci-mgmt` workflow run as the starting point branch.